ahds/__init__.py

```diff
@@ -1,30 +1,47 @@
 # -*- coding: utf-8 -*-
 # ahds
 
-import header
-import data_stream
+try:
+    from .header import AmiraHeader
+except:
+    from header import AmiraHeader
+try:
+    from .data_stream import DataStreams
+except:
+    from data_stream import DataStreams
+try:
+    from .ahds_common import deprecated
+except:
+    from ahds_common import deprecated
 
+import os.path as path
 
-class AmiraFile(object):
+
+class AmiraFile(AmiraHeader):
     """Convenience class to handle Amira files
     
-    This class aggregates user-level classes from the :py:mod:`ahds.header` and :py:mod:`ahds.data_stream` modules
-    into a single class with a simple interface :py:meth:`AmiraFile.header` for the header and :py:attr:`AmiraFile.data_streams` 
-    data streams attribute.
+    This class is a user-level alias classe for the :py:class:`ahds.header.AmiraHeader` and 
+    binds it to the deprecated :py:class: `ahds.data_stream.DataStreams` class.
+    the latter is marked deprecated and will be removed in future. As a consequence
+    the AmiraHeader class will be merged into AmiraFile class and will be marked deprecated.
     """
+
+    __slots__ = ("_data_streams",)
     def __init__(self, fn, *args, **kwargs):
-        self._fn = fn
-        self._header = header.AmiraHeader.from_file(self._fn, *args, **kwargs)
-        self._data_streams = None # only populate on call to read() method
-        
+        super(AmiraFile,self).__init__(fn,*args,**kwargs)
```

Comment:
I think there will be a signature mismatch here between this class and its parent.

Reply:
lets check when proper tests are in place. In most cases only file name is needed.